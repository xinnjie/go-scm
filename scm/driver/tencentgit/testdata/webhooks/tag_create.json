{
  "object_kind": "tag_push",
  "operation_kind": "create",
  "ref": "refs/tags/v1.0.2",
  "before": "1480a4610ca01dd10cb5bc3359a1b1ea568c09a1",
  "after": "b96850262fabfa9a1d9d28fff9040621958379f9",
  "user_id": 11323,
  "user_name": "git_user1",
  "project_id": 11452,
  "repository": {
    "name": "tencent",
    "url": "ssh://git@tencent.com/z-413/tencent.git",
    "description": "",
    "homepage": "https://tencent.com/z-413/tencent",
    "git_http_url":"http://tencent.com/z-413/tencent.git",
    "git_ssh_url":"git@tencent.com:z-413/tencent.git",
    "visibility_level":0
  },
  "commits":[
    {
      "id":"458ce39e1a28572597979ca6c7cdc6c338f6bd43",
      "message":"update",
      "timestamp":"2019-03-28T02:14:59+0000",
      "url":"https://git.example.tencent.com/release-test/xxx-framework/commit/458ce39e1a28572597979ca6c7cdc6c338f6bd43",
      "author":{
        "name":"xuhaohe",
        "email":"xuhaohe@tencent.com"
      },
      "added":[],
      "modified":[
        "README.md"
      ],
      "removed":[]
    }
  ],
  "total_commits_count":1,
  "create_from":"master"
}