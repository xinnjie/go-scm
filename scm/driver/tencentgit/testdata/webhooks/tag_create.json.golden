{
    "Ref": "refs/tags/v1.0.2",
    "After": "b96850262fabfa9a1d9d28fff9040621958379f9",
    "Repo": {
        "ID": "11452",
        "Namespace": "z-413",
        "Name": "tencent",
        "FullName": "z-413/tencent",
        "Perm": null,
        "Branch": "",
        "Private": false,
        "Clone": "http://tencent.com/z-413/tencent.git",
        "CloneSSH": "git@tencent.com:z-413/tencent.git",
        "Link": "https://tencent.com/z-413/tencent",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    },
    "Commit": {
        "Sha": "458ce39e1a28572597979ca6c7cdc6c338f6bd43",
        "Message": "update",
        "Author": {
            "Name": "xuhaohe",
            "Email": "xuhaohe@tencent.com",
            "Date": "0001-01-01T00:00:00Z",
            "Login": "xuhaohe",
            "Avatar": ""
        },
        "Committer": {
            "Name": "git_user1",
            "Email": "",
            "Date": "0001-01-01T00:00:00Z",
            "Login": "git_user1",
            "Avatar": ""
        },
        "Link": "https://git.example.tencent.com/release-test/xxx-framework/commit/458ce39e1a28572597979ca6c7cdc6c338f6bd43"
    },
    "Sender": {
        "Login": "git_user1",
        "Name": "git_user1",
        "Email": "",
        "Avatar": ""
    }
}
